app/src/main/java/id/rizmaulana/covid19/ui/overview/DashboardActivity.kt

```diff
@@ -109,6 +117,24 @@ class DashboardActivity : BaseActivity() {
 
     }
 
+    private fun handlePinnedUpdate(data: CovidDetail?) {
+        with(binding.countryInfo) {
+            data?.let {detail ->
+                val header = StringBuilder().apply {
```

Comment:
Is it to get proper location name? we can use `detail.locationName`
```
val locationName get() = countryRegion + if (!provinceState.isNullOrEmpty()) ", $provinceState" else ""
```

Reply:
oh didnt you wrote this, will refactor